snap/wrappers/juju

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export LXD_DIR=/var/snap/lxd/common/lxd
```

Comment:
Yes, as it stands this uses the LXD snap, which has a different socket. The two LXD's do NOT see the same containers.

That said, @wallyworld and I were able to produce a condition on our machines that resulted in the 2 LXD's not being happy co-installed (at least, for juju's sake, as it reported it couldn't bind). Having juju and LXD installed as both a snap and a deb works fine for @battlemidget, and mirrors conjure-up, so it's odd to see the issue.